packages/web/src/services/audius-sdk/audiusSdk.ts

```diff
@@ -44,6 +44,21 @@ const initSdk = async () => {
   // Overrides some DN configuration from optimizely
   const discoveryNodeSelector = await discoveryNodeSelectorService.getInstance()
 
+  const message = `signature:${new Date().getTime()}`
```

Comment:
Note that this is a potential footgun because we sign once for a user and if happen to switch to other users it may be a problem